### PR TITLE
Disable hw accl

### PR DESCRIPTION
**Pull request by aaomidi**
Disables the GPU process and any hardware acceleration

### CODE REVIEW

**Comment by anthonydresser** [comment]
This option already exists i think. https://github.com/microsoft/azuredatastudio/blob/master/resources/completions/zsh/_code#L34

File: src/main.js [hunk 1]
```diff
@@ -27,6 +27,11 @@ bootstrap.enableASARSupport();
 
 // Set userData path before app 'ready' event and call to process.chdir
 const args = parseCLIArgs();
+// {{SQL CARBON EDIT}}
```
Comment (anthonydresser) [inline src/main.js]: Inline comments are better.

**Comment by aaomidi** [comment]
Looks like electron has completely messed up the order of these being executed.

**Comment by aaomidi** [comment]
https://github.com/electron/electron-packager/issues/418

**Comment by anthonydresser** [comment]
Does this fix the tests? Seems weird that this would cause an issue? I'm assuming the containers don't have access to a gpu, so electron wouldn't try and use one.

**Comment by aaomidi** [comment]
> Does this fix the tests? Seems weird that this would cause an issue? I'm assuming the containers don't have access to a gpu, so electron wouldn't try and use one.

Not yet, but it seems to have some impacts on it.

**Comment by alanrenmsft** [comment]
trying to fix the display issue? can you provide more context please?

**Comment by aaomidi** [comment]
This doesn't just fix the display issue but allows the program to run in headless mode. This is pretty great for testing as it decreases the test time. I'm in the progress of testing this with my new PR validation pipeline to see how it performs.

**Pull request merged**